src/patterns/molecules/sticky-promo/sticky-promo.hbs

```diff
@@ -0,0 +1,22 @@
+---
+name: Sticky Promo Card
+description: A sticky promo card promotes content,
+notes: |
+  - Use only one primary cta per card.
+  - Use only one sticky promo card per page.
+  - The copy on the card should be short and succinct, ideally no longer than 2 lines at display size in English.
+  - Always include a product logo in the card content.
```

Comment:
If I do omit the logo class the content can slightly overlap the close button. Maybe we should increase the padding to prevent that, and could also nudge the close button a little further into the corner.

![image](https://user-images.githubusercontent.com/205591/88210578-443f6a00-cc09-11ea-8cf8-1909ca8dc0d8.png)

Reply:
[The specs that kropp gave me](https://docs.google.com/document/d/1PM3PHlai77s5Xj9OF0bRqbSGv_8JdI8QnGT3RTfQb7E/edit?usp=sharing) say that the card should
> Always include a product logo in the card content

Reply:
Since this is no longer required we should change this line. Something like:
```
- Add one of these product theme classes to include a logo above the title:
  - `mzp-t-product-firefox`
  - `mzp-t-product-beta`
  - `mzp-t-product-developer`
  - `mzp-t-product-nightly`
```